src/components/FormCard.tsx

```diff
@@ -10,22 +10,14 @@ import * as Antd from 'antd';
 import FormFieldSet from '../building-blocks/FormFieldSet';
 import FormManager from '../utilities/FormManager';
 import { fillInFieldSets } from '../utilities/common';
-import { ICommonCardProps, IFieldSet } from '../interfaces';
-import { IForm } from '../props';
-
-export interface IFormCardProps extends ICommonCardProps {
-  children?: any;
-  defaults?: object;
-  isLoading?: boolean;
-  model?: any;
-  onCancel?: () => void;
-  onSave: (data: object) => Promise<void>;
-  renderTopRight?: () => any;
-}
+import { IFieldSet } from '../interfaces';
+import { IFormProps, IWrappedFormProps } from '../props';
 
-interface IFormCardWrappedProps extends IFormCardProps {
-  form: IForm;
-}
+import { ICardProps } from './Card';
+
+export interface IFormCardProps extends ICardProps, IFormProps {}
```

Comment:
It makes more sense to extend in `IFormProps, ICardProps` order, since it is `IFormCard`

Reply:
I mean... 🤷‍♂️sure?